### PR TITLE
meta: add some config files to pbxproj for easy editing and clean up the list

### PR DESCRIPTION
**Pull request by armcknight**
#skip-changelog

### CODE REVIEW

File: Sentry.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1633,7 +1639,6 @@
 		844DA80F28246D5000E6B62E /* README.md */ = {isa = PBXFileReference; lastKnownFileType = net.daringfireball.markdown; path = README.md; sourceTree = "<group>"; };
 		844DA81028246D5000E6B62E /* CONTRIBUTING.md */ = {isa = PBXFileReference; lastKnownFileType = net.daringfireball.markdown; path = CONTRIBUTING.md; sourceTree = "<group>"; };
 		844DA81628246D5000E6B62E /* .gitignore */ = {isa = PBXFileReference; lastKnownFileType = text; path = .gitignore; sourceTree = "<group>"; };
-		844DA81B28246D9300E6B62E /* Brewfile.lock.json */ = {isa = PBXFileReference; lastKnownFileType = text.json; path = Brewfile.lock.json; sourceTree = "<group>"; };
```
Comment (armcknight) [inline Sentry.xcodeproj/project.pbxproj:1636]: We don't track this in git anymore

File: Sentry.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1675,7 +1680,6 @@
 		84C47B2B2A09239100DAEB8A /* .codecov.yml */ = {isa = PBXFileReference; lastKnownFileType = text.yaml; path = .codecov.yml; sourceTree = "<group>"; };
 		84DEE86A2B686BD400A7BC17 /* SentrySamplerDecision.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; name = SentrySamplerDecision.h; path = include/SentrySamplerDecision.h; sourceTree = "<group>"; };
 		84DEE8752B69AD6400A7BC17 /* SentryLaunchProfiling.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; name = SentryLaunchProfiling.h; path = Sources/Sentry/include/SentryLaunchProfiling.h; sourceTree = SOURCE_ROOT; };
-		84E4F5692914F020004C7358 /* Brewfile */ = {isa = PBXFileReference; indentWidth = 2; lastKnownFileType = text; path = Brewfile; sourceTree = "<group>"; tabWidth = 2; };
```
Comment (armcknight) [inline Sentry.xcodeproj/project.pbxproj:1678]: This was in here twice

File: Sentry.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -3309,13 +3316,6 @@
 			path = Profiling;
 			sourceTree = "<group>";
 		};
-		84281C552A579C2B00EE88F2 /* SentryTestUtilsObjc */ = {
-			isa = PBXGroup;
-			children = (
-			);
-			path = SentryTestUtilsObjc;
-			sourceTree = "<group>";
-		};
```
Comment (armcknight) [inline Sentry.xcodeproj/project.pbxproj:3318]: This has been a zombie for a while

**Review by philipphofmann — APPROVED** [review]
Thanks, LGTM 👍

**Review by brustolin — COMMENTED** [review]
Can we have an Xcode group (folder) to hide all these configuration files? 
It's annoying that they take up a large portion of the project tree, yet are the least accessed files.

**Comment by armcknight** [comment]
> Can we have an Xcode group (folder) to hide all these configuration files? It's annoying that they take up a large portion of the project tree, yet are the least accessed files.

I wouldn't be opposed if someone wants to do that, but I don't feel like taking it on as it will require path changes in lots of config code. I exclusively use ⌘ ⇧ O to navigate to source files so the list doesn't bother me; all I care about is that they are editable from inside Xcode at all.

**Pull request merged**